additional/docs/blog/posts/vcmi-1.5.0-release.md

```diff
@@ -14,16 +14,34 @@ A new major release has been published. VCMI 1.5.0 sees the light of day. The ma
 ## Online multiplayer support
 VCMI now has an online lobby that can be used to set up multiplayer sessions. But you can also interact with other players. The lobby works on all platforms and regardless of firewall restrictions. It can be opened at any time on desktop systems with `CTRL + Tab` and on mobile platforms with a three-finger touch.
 
-## More natural looking random maps
+## Better random maps
 A new biome system for random maps has been integrated. This makes created maps look much more natural.
 
+Also there are many different optimizations like properly random-looking zone edges, treasure/obstacle density tweaks and underground generation content tweaks leading to better zones.
+
+## AI optimizations
+Nullkiller AI got notable speedup improvements, better handling of heroes with "patrol" set in map editor and some other AI bug fixes.
```

Comment:
I would simply change to "some other fixes", since the stuff in previous part of sentence were not really bug fixes so  word"other" doesnt fit etc.